### PR TITLE
#10340

### LINKED ISSUE #10340: Create the KeyCredential Admins Security Global Group?
Is this step even necessary on Windows Server 2019? Server 2019 by default already has a 'Key Admins' group setup. Investigation is needed..

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 13c400ea-f52d-b45b-a871-31c94262084d
* Version Independent ID: 9465e043-990f-cb8b-2d4b-33dc8d3c21bf
* Content: [Update Active Directory schema for cert-trust deployment (Windows Hello for Business) - Windows security](https://docs.microsoft.com/en-us/windows/security/identity-protection/hello-for-business/hello-cert-trust-validate-ad-prereq)
* Content Source: [windows/security/identity-protection/hello-for-business/hello-cert-trust-validate-ad-prereq.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/public/windows/security/identity-protection/hello-for-business/hello-cert-trust-validate-ad-prereq.md)
* Product: **m365-security**
* GitHub Login: @mapalko
* Microsoft Alias: **mapalko**

### PR DESCRIPTION
**Pull request by michaelAngeloEgypt**
fixes #10340 
the feedback was about stressing that a step is not needed for Windows Server 2019. I discovered that this is already mentioned in the article, so I made that statement bold to make it stand out.

### CODE REVIEW

File: windows/security/identity-protection/hello-for-business/hello-cert-trust-validate-ad-prereq.md [hunk 1]
```diff
@@ -25,7 +25,9 @@ ms.reviewer:
 - On-premises deployment
 - Certificate trust
 
-The key registration process for the On-premises deployment of Windows Hello for Business needs the Windows Server 2016 Active Directory or later schema.  The key-trust model receives the schema extension when the first Windows Server 2016 or later domain controller is added to the forest.  The certificate trust model requires manually updating the current schema to the Windows Server 2016 or later schema. If you already have a Windows Server 2016 or later domain controller in your forest, you can skip the **Updating the Schema** and **Create the KeyCredential Admins Security Global Group** steps.
+The key registration process for the On-premises deployment of Windows Hello for Business needs the Windows Server 2016 Active Directory or later schema.  The key-trust model receives the schema extension when the first Windows Server 2016 or later domain controller is added to the forest.  The certificate trust model requires manually updating the current schema to the Windows Server 2016 or later schema.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-cert-trust-validate-ad-prereq.md:28]:
```suggestion
The key registration process for the on-premises deployment of Windows Hello for Business needs the Windows Server 2016 Active Directory or later schema. The key-trust model receives the schema extension when the first Windows Server 2016 or later domain controller is added to the forest. The certificate trust model requires manually updating the current schema to the Windows Server 2016 or later schema. 
```

File: windows/security/identity-protection/hello-for-business/hello-cert-trust-validate-ad-prereq.md [hunk 2]
```diff
@@ -25,7 +25,9 @@ ms.reviewer:
...
+The key registration process for the On-premises deployment of Windows Hello for Business needs the Windows Server 2016 Active Directory or later schema.  The key-trust model receives the schema extension when the first Windows Server 2016 or later domain controller is added to the forest.  The certificate trust model requires manually updating the current schema to the Windows Server 2016 or later schema. 
+
+**If you already have a Windows Server 2016 or later domain controller in your forest, you can skip the 'Updating the Schema' and 'Create the KeyCredential Admins Security Global Group' steps below.**
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-cert-trust-validate-ad-prereq.md:30]:
```suggestion
> [!NOTE]
> If you already have a Windows Server 2016 or later domain controller in your forest, you can skip the "Updating the Schema" and "Create the KeyCredential Admins Security Global Group" steps that follow.**
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @michaelAngeloEgypt, kindly commit the suggested changes. Many thanks.

File: windows/security/identity-protection/hello-for-business/hello-cert-trust-validate-ad-prereq.md [hunk 3]
```diff
@@ -25,7 +25,10 @@ ms.reviewer:
...
+The key registration process for the on-premises deployment of Windows Hello for Business needs the Windows Server 2016 Active Directory or later schema. The key-trust model receives the schema extension when the first Windows Server 2016 or later domain controller is added to the forest. The certificate trust model requires manually updating the current schema to the Windows Server 2016 or later schema. 
+
+> [!NOTE]
+> If you already have a Windows Server 2016 or later domain controller in your forest, you can skip the "Updating the Schema" and "Create the KeyCredential Admins Security Global Group" steps that follow.**
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-cert-trust-validate-ad-prereq.md:31]:
```suggestion
> If you already have a Windows Server 2016 or later domain controller in your forest, you can skip the "Updating the Schema" and "Create the KeyCredential Admins Security Global Group" steps that follow.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @michaelAngeloEgypt, one more cosmetic change, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by aczechowski — APPROVED** [review]

**Pull request merged**